content/rc/rc-quickstart.md

```diff
@@ -132,24 +132,23 @@ Docker provides a convenient way to run `redis-cli` without the full installatio
 
 When you run the `redis` Docker image, you can open a bash shell instance and run `redis-cli` from the container.
 
-1.  To begin, pull the `redis` docker image and run it with default options:
+1.  Enter this command in your command-line interface. A bash prompt opens in your Docker image.
```

Comment:
```suggestion
1.  Run the following commands to create a Redis Docker container and connect to a bash prompt within the container.
```

Reply:
@rrelledge this context was provided in the intro para. Consider applying your edit there for precision.

Reply:
I think one of the issues it that set of commands actually does three things:
1. Downloads the Redis Docker image (docker pull)
2. Starts a container created from the image (docker run)
3. Connects to a bash prompt running in the container (docker exec -it redis1 bash)

The previous instruction sounds like that set of commands only opens a bash prompt.

Reply:
Then I feel that needs to be part of the context-setting paragraph. The step should just be user input + expected result.